pkg/virt-launcher/virtwrap/util/libvirt_helper.go

```diff
@@ -368,3 +368,18 @@ func SetupLibvirt() error {
 
 	return nil
 }
+
+func GetInterfaceType(iface *v1.Interface) string {
```

Comment:
Is it a leftover? I see another is defined in pkg/virt-launcher/virtwrap/api/defaults.go

Reply:
There's a import cycle that is hard to untangle hence the duplication (which is btw not part of this PR - the duplicated code was just moved from inline to here).

Reply:
I just don't see where is it being used... 
I only see a call in the convertor and in podInterface `api.GetInterfaceType(s.iface)`
I can't see where the one in utils is being called.